package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "cldr-data": ">=25",
     "globalize": "^1.1.0-rc.5 <1.3.0",
-    "webpack": "^2.5.1"
+    "webpack": "^3.0.0"
```

Comment:
> After much reading about wp3, there isn't much about it so a 1.1 seems more than enough if you ask me ;)

In practice, I think you are right, but in theory this is an issue. Given webpack did a major bump, it can introduce breaking changes. If we handle both simultaneously, we'll eventually introduce if webpack-2 do this, if webpack-3 do that, which make maintenance and testing harder.